src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -109,36 +206,36 @@ private static void runTabulationTest(String stem, String expectedException) {
     List<String> exceptionsEncountered = session.tabulate();
     if (expectedException != null) {
       assertTrue(exceptionsEncountered.contains(expectedException));
-      return;
-    }
-    Logger.info("Examining tabulation test results...");
-    String timestampString = session.getTimestampString();
-    ContestConfig config = ContestConfig.loadContestConfig(configPath);
-    assertNotNull(config);
+    } else {
+      Logger.info("Examining tabulation test results...");
+      String timestampString = session.getTimestampString();
+      ContestConfig config = ContestConfig.loadContestConfig(configPath);
+      assertNotNull(config);
+
+      if (config.isMultiSeatSequentialWinnerTakesAllEnabled()) {
+        for (int i = 1; i <= config.getNumberOfWinners(); i++) {
+          compareFiles(config, stem, timestampString, Integer.toString(i));
+        }
+      } else {
+        compareFiles(config, stem, timestampString, null);
+      }
 
-    if (config.isMultiSeatSequentialWinnerTakesAllEnabled()) {
-      for (int i = 1; i <= config.getNumberOfWinners(); i++) {
-        compareFiles(config, stem, timestampString, Integer.toString(i));
+      // If this is a Dominion tabulation test, also check the converted output file.
```

Comment:
Does it still make sense to limit this to Dominion in light of #715?

Reply:
Nope -- removed since #715 merged first!